Presentation.Web/app/components/org/structure/org-unit-migration-table.component.ts

```diff
@@ -0,0 +1,159 @@
+﻿module Kitos.Organization.Structure.Components {
+    "use strict";
+
+    function setupComponent(): ng.IComponentOptions {
+        return {
+            bindings: {
+                title: "@",
+                options: "<",
+                configuration: "<",
+                unitUuid: "@",
+                organizationUuid: "@"
+            },
+            controller: OrganizationUnitMigrationTableController,
+            controllerAs: "ctrl",
+            templateUrl: "app/components/org/structure/org-unit-migration-table.view.html"
+        }
+    }
+
+    export enum MigrationTableColumnType {
+        Text = 0,
+        Link = 1
+    }
+
+    export interface IMigrationTableColumn{
+        title: string;
+        property: string;
+        type: MigrationTableColumnType;
+        cssClass?: string;
+    }
+
+    interface IOrganizationUnitMigrationTableController extends ng.IComponentController {
+        title: string;
+        options: IOrganizationUnitMigrationOptions;
+        configuration: IMigrationTableColumn[];
+        unitUuid: string;
+        organizationUuid: string;
+    }
+
+    class OrganizationUnitMigrationTableController implements IOrganizationUnitMigrationTableController {
+        title: string | null = null;
+        options: IOrganizationUnitMigrationOptions | null;
```

Comment:
remember to set to null if | null .. otherwise it will be undefined and undefined !== null since you use null in your checks